spec/install/gemfile/platform_spec.rb

```diff
@@ -230,21 +230,46 @@
     expect(out).not_to match(/Could not find gem 'some_gem/)
   end
 
-  it "prints a helpful warning when a dependency is unused on any platform" do
-    simulate_platform "ruby"
-    simulate_ruby_engine "ruby"
+  context "when disable_platform_warnings is false (by default)" do
```

Comment:
This test doesn't need a `context` blocked wrapping around it as the setting is set in Bundler automatically.